skaffold.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: skaffold/v1
+kind: Config
+metadata:
+  name: conference
+build:
+  artifacts:
+  - image: sanek555/api-gateway
+    context: sa-gateway
+  - image: sanek555/sponsorship
+    context: sponsorship
+  - image: conference
```

Comment:
strange image name

Reply:
This is just a placeholder to build an image from Dockerfile. I suppose, the correct name will be provided later (@ilya-a87 ?)